requirements.txt

```diff
@@ -0,0 +1,2 @@
+ijson==3.1.4
+requests==2.28.1
```

Comment:
There are a number of ways to manage dependencies in python, but I think the simplest and easiest is the [requirements file](https://pip.pypa.io/en/stable/reference/requirements-file-format/). Having some way of listing your dependencies means someone can clone the repo and start working immediately without having to leave your repo.